src/app/modules/submission/email-submission/ParsedResponsesObject.class.ts

```diff
@@ -1,11 +1,9 @@
 import { err, ok, Result } from 'neverthrow'
 
+import { IFormDocument } from 'src/types'
+
 import { FormAuthType, FormResponseMode } from '../../../../../shared/types'
-import {
-  FieldResponse,
-  FormFieldSchema,
-  IFormDocument,
-} from '../../../../types'
+import { FieldResponse, FormFieldSchema } from '../../../../types'
```

Comment:
nit: should we move `IFormDocument` import in line 3 to line 6 since both the relative and absolute imports are pointing to the same file directory?